core/src/main/java/greencity/controller/OrderController.java

```diff
@@ -530,4 +531,25 @@ public ResponseEntity<List<LocationsDto>> getAllLocationsByCourierId(
         List<LocationsDto> locations = ubsClientService.getAllLocationsByCourierId(courierId);
         return ResponseEntity.status(HttpStatus.OK).body(locations);
     }
+
+    /**
+     * Receives and processes payment information from the Monobank API. This method
+     * handles the incoming payment response and validates the payment details
+     * provided by Monobank. The payment details are logged and then passed to the
+     * {@code validatePaymentFromMonoBank} method for further validation and
+     * processing.
+     *
+     * @param response the payment response received from Monobank, containing
+     *                 details such as transaction ID, status, and amount.
+     */
+    @Operation(summary = "Receive payment information from Monobank API")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = HttpStatuses.OK),
+        @ApiResponse(responseCode = "400", description = HttpStatuses.BAD_REQUEST)
+    })
+    @PostMapping("/monobank/payments")
+    public void receivePaymentFromMonoBank(@RequestBody MonoBankPaymentResponseDto response) {
```

Comment:
based on the method name if you call it you expect to `receive` something. Why return type is `void`?

Reply:
This method receives a response from the MonoBank API, checks the response in the current endpoint, and validates the payment if the payment status is successful. No responses need to be sent back to the API.

Reply:
`validates the payment if the payment status is successful`
What does happen if the response is not successful?